### PR TITLE
PYIC-5150: Add stub data evidence for the DWP KBV stub

### PR DESCRIPTION
**Pull request by AmritSidhu**
### What changed

Add stub data evidence for the DWP KBV stub

### Issue tracking

- [PYIC-5150](https://govukverify.atlassian.net/browse/PYIC-5150)

### Environment variables or secrets

- [x] No environment variables or secrets were added or changed

[PYIC-5150]: https://govukverify.atlassian.net/browse/PYIC-5150?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json [hunk 1]
```diff
@@ -2395,6 +2395,182 @@
...
     },
+    {
+      "criType": "DWP KBV (Stub)",
+      "label": "Alice Parker (Valid) DWP KBV",
+      "payload": {
+        "name": [
+          {
+            "nameParts": [
+              {
+                "value": "Aliçe",
```
Comment (MikeCollingwood) [inline di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json:2406]: Are the "c" here and the "e" in parker intentionally special? I couldn't see it other examples

File: di-ipv-credential-issuer-stub/src/main/resources/data/criStubEvidencePayloads.json [hunk 2]
```diff
@@ -408,6 +408,58 @@
...
+    {
+      "criType": "DWP KBV (Stub)",
+      "label": "Failed DWP KBV (Stub)",
+      "payload": {
+        "type": "IdentityCheck",
+        "txn": "8KJTLGN7QX",
+        "verificationScore": 0,
+        "ci": ["V03"],
+        "failedCheckDetails": [{
+					"kbvResponseMode": "free_text",
```
Comment (MikeCollingwood) [inline di-ipv-credential-issuer-stub/src/main/resources/data/criStubEvidencePayloads.json:446]: v minor but could we reduce the indent

File: di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json (hunk 1 above)
Comment (AmritSidhu) [reply di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json:2406]: This data was duplicated from the HMRC KBV data above it. Looks like the Experian KBV / NINO / BAV use it with this "ç" as well

File: di-ipv-credential-issuer-stub/src/main/resources/data/criStubEvidencePayloads.json (hunk 2 above)
Comment (AmritSidhu) [reply di-ipv-credential-issuer-stub/src/main/resources/data/criStubEvidencePayloads.json:446]: In the editor this is fine, no crazy indentation, I think its just GitHub displaying it weirdly

File: di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json (hunk 1 above)
Comment (blakeyp) [reply di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json:2406]: I think this is deliberate to test name correlation 😄

**Review by blakeyp — APPROVED** [review]

**Pull request merged**